pkg/server/api_v2_databases_metadata.go

```diff
@@ -1068,6 +1086,27 @@ func (a *apiV2Server) updateTableMetadataJobAuthorized(
 	return count > 0, nil
 }
 
+func checkVersion(ctx context.Context, w http.ResponseWriter, settings *cluster.Settings) bool {
+	if !settings.Version.IsActive(context.TODO(), clusterversion.V24_3_AddTableMetadataCols) {
+		message := "This API is not accessible on this version of CockroachDB"
```

Comment:
WDYT about adding additional instruction on finalizing the cluster? e.g.
```
This API is not accessible on this version of CockroachDB, please finalize your upgrade if it is in progress.
```

Reply:
I think that is a message more suited for the front end to display to the user and not for the API to return to the client.